### DISCUSSION TITLE
Ability to add styles for multiple tags in CK Editor

### DISCUSSION THREAD

**Discussion by patrikalienus** [craftcms/ckeditor#291] [Ideas] [answered]
I need the ability to add the class "uppercase" to any p or h1-h6 tag in CK Editor.
This results in 7 separate styles which is very cumbersome to maintain in the long run, especially considering this is just one example.

If there is a way to make 1 style enabled for multiple HTML tags, I haven't found it.

Edit; This feels like the wrong place for this feature request, but it's where I ended up from the craftcms/ckeditor repo. I also realise this might be a limitation of CK Editor, but Drupal have (apparently) solved this for CK Editor. I'm not using Drupal myself so I can't check though.

**Comment by brandonkelly** [comment, marked as answer]
This is a CKEditor limitation. There’s an open request to allow style definitions to target multiple block elements @ https://github.com/ckeditor/ckeditor5/issues/12771; feel free to voice your support over there!

**Reply by patrikalienus** [reply]
Ah ok thanks. I will join that discussion.
